com.chartboost.mediation.demo/Assets/Plugins/Android/mainTemplate.gradle

```diff
@@ -0,0 +1,137 @@
+    implementation 'com.mbridge.msdk.oversea:playercommon:16.0.31' // Assets/Samples/Chartboost Mediation/4.0.0/Mintegral/Editor/Optional-MintegralDependencies.xml:14
+    implementation 'com.mbridge.msdk.oversea:reward:16.0.31' // Assets/Samples/Chartboost Mediation/4.0.0/Mintegral/Editor/Optional-MintegralDependencies.xml:15
+    implementation 'com.mbridge.msdk.oversea:same:16.0.31' // Assets/Samples/Chartboost Mediation/4.0.0/Mintegral/Editor/Optional-MintegralDependencies.xml:17
+    implementation 'com.mbridge.msdk.oversea:videocommon:16.0.31' // Assets/Samples/Chartboost Mediation/4.0.0/Mintegral/Editor/Optional-MintegralDependencies.xml:16
+    implementation 'com.mbridge.msdk.oversea:videojs:16.0.31' // Assets/Samples/Chartboost Mediation/4.0.0/Mintegral/Editor/Optional-MintegralDependencies.xml:12
+    implementation 'com.pangle.global:ads-sdk:4.3.0.4' // Assets/Samples/Chartboost Mediation/4.0.0/Pangle/Editor/Optional-PangleDependencies.xml:12
+    implementation 'com.tapjoy:tapjoy-android-sdk:12.9.1' // Assets/Samples/Chartboost Mediation/4.0.0/TapJoy/Editor/Optional-TapJoyDependencies.xml:11
+    implementation 'com.unity3d.ads:unity-ads:4.4.1' // Assets/Samples/Chartboost Mediation/4.0.0/UnityAds/Editor/Optional-UnityAdsDependencies.xml:8
+    implementation 'com.vungle:publisher-sdk-android:6.12.0' // Assets/Samples/Chartboost Mediation/4.0.0/Vungle/Editor/Optional-VungleDependencies.xml:8
+    implementation 'com.yahoo.mobile.ads:android-yahoo-mobile-sdk:1.3.0' // Assets/Samples/Chartboost Mediation/4.0.0/Yahoo/Editor/Optional-YahooDependencies.xml:13
+    implementation 'org.greenrobot:eventbus:3.3.1' // Assets/Samples/Chartboost Mediation/4.0.0/Chartboost Mediation/Editor/ChartboostMediationDependencies.xml:11
+    implementation 'org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.7.20' // Assets/Samples/Chartboost Mediation/4.0.0/Google Bidding/Editor/Optional-GoogleBiddingDependencies.xml:12
+    // implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-android:1.6.0' // Assets/Samples/Chartboost Mediation/4.0.0/Google Bidding/Editor/Optional-GoogleBiddingDependencies.xml:11
+    implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-android:1.6.4' // Assets/Samples/Chartboost Mediation/4.0.0/Chartboost Monetization/Editor/Optional-ChartboostDependencies.xml:11
+// Android Resolver Dependencies End
+**DEPS**}
+
+// Android Resolver Exclusions Start
+android {
+  packagingOptions {
+      exclude ('/lib/armeabi/*' + '*')
+      exclude ('/lib/mips/*' + '*')
+      exclude ('/lib/mips64/*' + '*')
+      exclude ('/lib/x86/*' + '*')
+      exclude ('/lib/x86_64/*' + '*')
+  }
+}
+// Android Resolver Exclusions End
+android {
+    compileSdkVersion **APIVERSION**
+    buildToolsVersion '**BUILDTOOLS**'
+
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
```

Comment:
🤔 .... this is still needed on Unity, isn't?? I wonder when they will start using Java 11.